cedar-lean/UnitTest/CedarProto.lean

```diff
@@ -50,20 +50,6 @@ namespace UnitTest.CedarProto
 open CedarProto
 open Cedar.Data
 
-/--
-  `filename` is expected to be the name of a file containing binary protobuf data.
-  This test will ensure that that binary data deserializes into the value `expected`.
--/
-def testDeserializeProtodata [Inhabited α] [DecidableEq α] [Repr α] [Proto.Message α]
-  (filename : String) (expected : α) : TestCase IO :=
-  test s!"Deserialize {filename}" ⟨λ () => do
-    let buf ← IO.FS.readBinFile filename
-    let parsed : Except String α := Proto.Message.interpret? buf
-    match parsed with
-    | .ok req => checkEq req expected
-    | .error e => pure (.error e)
-  ⟩
-
 /--
   Similar to `testDeserializeProtodata`, but `f` is applied to the deserialized
   value before comparing to `expected`
```

Comment:
Needs to be updated since `testDeserializeProtodata` is now based on this function.